### PR TITLE
Notebook 4.2

### PR DESCRIPTION
**Pull request by malev**
Do not merge till `jupyter 4.2` is out.

@bollwyvl

### CODE REVIEW

File: conda.recipe/meta.yaml [hunk 1]
```diff
@@ -9,14 +9,11 @@ build:
...
     - anaconda-client
-    - nbsetuptools
     - python
     - setuptools
   run:
     - anaconda-client
-    - nb_config_manager
-    - nbsetuptools
-    - notebook
+    - notebook ==5.0.0
```
Comment (bollwyvl) [inline conda.recipe/meta.yaml:16]: Love it.

File: nb_anacondacloud/nbextension/handlers.py [hunk 2]
```diff
@@ -58,15 +57,3 @@ def post(self, **args):
         except errors.BinstarError as e:
             self.log.error(e)
             self.set_status(400, str(e))
-
```
Comment (bollwyvl) [inline nb_anacondacloud/nbextension/handlers.py:61]: Yeah, i like this moving out of here.

File: nb_anacondacloud/setup.py [hunk 3]
```diff
@@ -1,9 +0,0 @@
-from os.path import join, abspath, dirname
```
Comment (bollwyvl) [inline nb_anacondacloud/setup.py:1]: Woohoo!

**Comment by bollwyvl** [comment]
Just reproduced the build locally.

There is something I was doing wrong when calling the new install scripts on nbpresent that came over here. I'm looking into it...

**Comment by bollwyvl** [comment]
Oof... it seems like nb_anacondacloud might be publishing private by default now? Is this a thing I might have set?

**Comment by bollwyvl** [comment]
Ah, indeed: i had manually made my "untitled" package private, which ended being the same name (based on the notebook name).

**Comment by malev** [comment]
@bollwyvl if you already have a package in AS it will not change the permissions already setup. If you are creating a new package (notebook) it will be public by default

**Comment by damianavila** [comment]
Quick overview looks good, but I will review in depth tomorrow (still I have to finish reviewing the latest changes upstream)... btw, nice work guys!

**Comment by bollwyvl** [comment]
Went with #21 instead.

**Pull request closed without merging**